client/landing/jetpack-cloud/sections/settings/host-selection/index.tsx

```diff
@@ -50,7 +50,7 @@ const HostSelection: FunctionComponent = () => {
 		<div className="host-selection">
 			<div className="host-selection__notice">
 				{ translate(
-					'In order to restore your site, should something go wrong, you’ll need to provide your websites {{strong}}SSH{{/strong}}, {{strong}}SFTP{{/strong}} or {{strong}}FTP{{/strong}} server credentials. We’ll guide you through it:',
+					'In order to restore your site, should something go wrong, you’ll need to provide your website’s {{strong}}SSH{{/strong}}, {{strong}}SFTP{{/strong}} or {{strong}}FTP{{/strong}} server credentials. We’ll guide you through it:',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.